Feature: Add a cloudwatch ping

kellyje opened the pull request:
DispatchMe/meteor-app#1401

Commit:
Add a cloudwatch ping

jperl commented on ping.js:
```diff
@@ -0,0 +1,10 @@
+
+var DEFAULT_PING_INTERVAL = 60000; // 1m
+
+var settings = Meteor && Meteor.settings && Meteor.settings.public || {};
```
`Meteor.settings.public` should always be there, so you do not need all this protection. But this should prob just be on `Meteor.settings` since it is only on the server.

jperl commented on ping.js:
```diff
@@ -0,0 +1,10 @@
+var settings = Meteor && Meteor.settings && Meteor.settings.public || {};
+var pingInterval = settings.pingInterval || DEFAULT_PING_INTERVAL;
+
+Meteor.setInterval(function () {
```
Instead of pinging by default, I would only ping if the `pingInterval` is set

Commit:
Remove defaults & cleanup settings

Remove default interval & move interval out of public.
Only ping if interval is set.
Remove unneeded object protection.

Minor: Cleanup formatting.

jperl commented:
:+1:
